File: Server/src/commonStructures/networkObjectDropAttributes.ts
import Position from "./position";

/**
 * The informations about a dropped object
 */
export default class NetworkObjectDropAttributes {
    /**
     * The new position of the object.
     */
    public newPosition:Position;
    /**
     * The id of the object.
     */
    public objectID:number;
    /**
     * The constructor for the NetworkObjectDropAttributes class.
     * @param newPosition The new position of the object.
     * @param objectID The id of the object.
     */
    constructor(newPosition:Position, objectID:number) {
        this.newPosition = newPosition;
        this.objectID = objectID;
    }
}

File: Assets/Scripts/Network/CommonStructures/NetworkObjectDropAttributes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Network
{
    /// <summary>
    /// The informations about a dropped object
    /// </summary>
    public class NetworkObjectDropAttributes
    {
        /// <summary>
        /// The new position of the object.
        /// </summary>
        public Position newPosition;
        /// <summary>
        /// The id of the object.
        /// </summary>
        public int objectID;
    }
}
